dashboard/config/linux/upstream-apparmor-kasan-base.config

```diff
@@ -245,12 +244,12 @@ CONFIG_LD_ORPHAN_WARN_LEVEL="warn"
 CONFIG_SYSCTL=y
 CONFIG_HAVE_UID16=y
 CONFIG_SYSCTL_EXCEPTION_TRACE=y
+# CONFIG_SYSFS_SYSCALL is not set
```

Comment:
Let's enable this syscall. It was enabled, and it's enabled in Debian.

Reply:
Enabled